### DISCUSSION TITLE
Autocmd for post-select

### DISCUSSION THREAD

**Discussion by mosheavni** [ibhagwan/fzf-lua#1532] [Q&A] [answered]
I want to add a callback function to my `require('fzf-lua').git_branches()` that does something AFTER the fzf window has closed.
What are my options?

use-case, I want to reload my fugitive.vim after I switch a branch:

```lua
M.reload_fugitive_index = function()
  for _, buf in ipairs(vim.api.nvim_list_bufs()) do
    local bufname = vim.api.nvim_buf_get_name(buf)
    if vim.startswith(bufname, 'fugitive://') and string.find(bufname, '.git//') then
      vim.api.nvim_buf_call(buf, function()
        vim.cmd.edit() -- refresh the buffer
      end)
    end
  end
end
```

**Comment by ibhagwan** [comment, marked as answer]
All action keybinds happen after the interface has closed 
https://github.com/ibhagwan/fzf-lua/wiki#custom-actions
